app/views/organizations/_form.html.erb

```diff
@@ -14,6 +14,17 @@
     </label>
   </div>
 
+  <% if organization.persisted? %>
+    <hr />
+
+    <h3>Item information</h3>
+    <%= form.hidden_field 'normalization_steps[0][destination_tag]', maxlength: 3, value: organization.normalization_steps.values.dig(0, 'destination_tag') || 998 %>
+    <%= form.text_field 'normalization_steps[0][source_tag]', maxlength: 3, value: organization.normalization_steps.values.dig(0, 'source_tag'), label: 'What MARC field contains item-level information?' %>
+    <%= form.text_field 'normalization_steps[0][subfields][i]', maxlength: 1, value: organization.normalization_steps.values.dig(0, 'subfields', 'i'), label: 'Local identifier subfield' %>
+    <%= form.text_field 'normalization_steps[0][subfields][a]', maxlength: 1, value: organization.normalization_steps.values.dig(0, 'subfields', 'a'), label: 'Call number subfield' %>
+    <%= form.text_field 'normalization_steps[0][subfields][m]', maxlength: 1, value: organization.normalization_steps.values.dig(0, 'subfields', 'm'), label: 'Local library' %>
```

Comment:
I recall pain around serialized data, form names, and URL params in the past, is that why we're jumping through the extra index in the json here (or are we doing this to leave ourselves open to adding more steps)?

Reply:
Yep, both. This way seemed to play nice with the json column type and rails, so we backed away slowly.